src/Send/SendScreen/SendScreen.stories.tsx

```diff
@@ -1,50 +1,43 @@
 import {action} from '@storybook/addon-actions'
 import {storiesOf} from '@storybook/react-native'
-import {BigNumber} from 'bignumber.js'
 import React from 'react'
 
-import {mockWallet} from '../../../storybook'
+import {balances, mockWallet} from '../../../storybook'
 import {SelectedWalletProvider} from '../../SelectedWallet'
-import type {TokenEntry} from '../../types'
+import {YoroiWallet} from '../../yoroi-wallets'
+import {SendProvider, useSendContext} from '../Context/SendContext'
 import {SendScreen} from './SendScreen'
 
 storiesOf('SendScreen', module)
   .addDecorator((story) => <SelectedWalletProvider wallet={mockWallet}>{story()}</SelectedWalletProvider>)
-  .add('Default', () => {
-    const selectedAsset: TokenEntry = {
-      networkId: 300,
-      identifier: '',
-      amount: new BigNumber(12344.00234523),
-    }
-
-    return (
-      <SendScreen
-        sendAll={false}
-        onSendAll={action('onSendAll')}
-        selectedTokenIdentifier={selectedAsset.identifier}
-        amount=""
-        setAmount={action('setAmount')}
-        receiver=""
-        setReceiver={action('setReceiver')}
-      />
-    )
-  })
-  .add('sendAll', () => {
-    const selectedAsset: TokenEntry = {
-      networkId: 300,
-      identifier: '',
-      amount: new BigNumber(12344.00234523),
-    }
-
-    return (
-      <SendScreen
-        sendAll={true}
-        onSendAll={action('onSendAll')}
-        selectedTokenIdentifier={selectedAsset.identifier}
-        amount=""
-        setAmount={action('setAmount')}
-        receiver=""
-        setReceiver={action('setReceiver')}
-      />
-    )
-  })
+  .add('Default', () => <SendScreenTest isSendAll={false} />)
+  .add('SendAll', () => <SendScreenTest isSendAll={true} />)
```

Comment:
prefer boolean prop shorthand
`<Component prop={true} />` -> `<Component prop />`

Reply:
do we need 2 different stories?

Reply:
@jemubm asked me about it, my thought was: while sending all is checked the amount must be disabled so it's a different screen state.